python/test/unit_distributed/runner.py

```diff
@@ -21,6 +21,7 @@
     import test_contexts_arbmpi
     import test_contexts_mpi4py
     import test_domain_decompositions
+    import simulator
```

Comment:
I don't see anything that matches this module name; should this line be here?

Reply:
That's supposed to be `test_simulator`

Reply:
And it's for completeness. AFAICT, the `try` clause will always fail unless there's an `options.py` file in the same directory as `test_*` files and the `runner.py` file, which isn't the case at the moment.